src/network/networkd-dhcp4.c

```diff
@@ -204,6 +204,9 @@ static int dhcp4_request_route(Route *in, Link *link) {
         if (route->quickack < 0)
                 route->quickack = link->network->dhcp_quickack;
 
+        route->initcwnd = link->network->dhcp_initcwnd;
+        route->initrwnd = link->network->dhcp_initrwnd;
```

Comment:
Ah, I forgot that we support semi-static route, that is, Gateway=dhcp4.
So, a route may be already configured something like that,
```
[Route]
Destination=...
Gateway=dhcp4
InitialCongestionWindow=10
InitialAdvertisedReceiveWindow=10
```
In such case, `route->initcwnd` and/or `route->initrwnd` are already set.
So, please check they are zero before assigning dhcp_initcwnd or so.
```c
if (route->initcwnd == 0)
        route->network->dhcp_initcwnd;
if (route->initrwnd == 0)
        route->network->dhcp_initrwnd;
```